### PR TITLE
Disco test for push notifications

### PR DESCRIPTION
**Pull request by jacekwegr**
Previously, disco was tested only for one type of host; now it's tested for real and virtual hosts.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/161132) / small_tests / 2f0a0fbbc9f28fd068bd75e51d222c8347ed35d5
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3926/161132/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3926/161132/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2023-01-10_14.46.19/lib.mongooseim.logs/run.2023-01-10_14.46.20/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/161241) / small_tests / 0c66f7aa33a446e3ef12273c7271edd83b6c6dff
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3926/161241/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3926/161241/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2023-01-11_08.27.28/lib.mongooseim.logs/run.2023-01-11_08.27.29/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/161296) / small_tests / 7d252203e0c033d772d12db1c533f8de474107b4
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3926/161296/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3926/161296/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2023-01-11_08.59.38/lib.mongooseim.logs/run.2023-01-11_08.59.38/suite.log.html)

File: big_tests/tests/push_integration_SUITE.erl [hunk 1]
```diff
@@ -45,8 +45,9 @@ basic_groups() ->
      {group, failure_cases_v3},
      {group, failure_cases_v2},
      {group, integration_with_sm_and_offline_storage},
-     {group, enhanced_integration_with_sm}
-    ].
+     {group, enhanced_integration_with_sm},
+     {group, disco}
+     ].
```
Comment (chrzaszcz) [inline big_tests/tests/push_integration_SUITE.erl:50]:
```suggestion
    ].
```

File: big_tests/tests/push_integration_SUITE.erl [hunk 2]
```diff
@@ -164,13 +172,22 @@ init_per_group(G, Config) ->
     end,
     C.
 
+end_per_group(disco, Config) ->
+    escalus:delete_users(Config),
+    Config;
```
Comment (chrzaszcz) [inline big_tests/tests/push_integration_SUITE.erl:177]: Just a note - there is no need to return Config in `end_per_*` callbacks - the return value is ignored.

**Review by chrzaszcz — APPROVED** [review]
Looks good :+1:

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/161492) / small_tests / af62463f3cf30b6070ec2fc307e28b2b11da1927
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3926/161492/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3926/161492/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2023-01-12_11.38.45/lib.mongooseim.logs/run.2023-01-12_11.38.45/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]

**Pull request merged**